src/node_crypto.cc

```diff
@@ -481,6 +519,17 @@ void SecureContext::Init(const FunctionCallbackInfo<Value>& args) {
       min_version = TLS1_2_VERSION;
       max_version = TLS1_2_VERSION;
       method = TLS_client_method();
+    } else if (strcmp(*sslmethod, "TLSv1_3_method") == 0) {
```

Comment:
This will need to be refactored a bit now that TLS 1.3 support has officially landed in core. Will fix in a rebase after landing